src/core/lib/event_engine/iomgr_engine/posix_poller.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#ifndef GRPC_CORE_LIB_EVENT_ENGINE_IOMGR_ENGINE_POSIX_POLLER_H
+#define GRPC_CORE_LIB_EVENT_ENGINE_IOMGR_ENGINE_POSIX_POLLER_H
+
+#include <grpc/support/port_platform.h>
+
+#include "src/core/lib/event_engine/poller.h"
+
+namespace grpc_event_engine {
+namespace iomgr_engine {
+
+// Interface for posix pollers
+class PosixPoller : public experimental::Poller {
+ public:
+  virtual ~Poller() = default;
+  virtual absl::Status Work(grpc_core::Duration timeout) = 0;
```

Comment:
Remove this and Kick, they're from `experimental::Poller`

Reply:
I've removed the file, it was speculative and Vignesh is working on the Posix version at the moment.